### ISSUE TITLE
Gray padding when rand_crop

### ISSUE THREAD

**Issue by jytime** [tensorflow/models#4320]
------------------------
### System information
- **What is the top-level directory of the model you are using**:deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04.2
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:1.7.0
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:CUDA Version 8.0.61
- **GPU model and memory**:two NVIDIA Corporation GM200 [GeForce GTX TITAN X]

### Describe the problem
Hi
When I am using deeplab v3 to train on SYNTHIA dataset (http://synthia-dataset.net/). This is a dataset with a similar format with cityscapes. It seems that some gray paddings are added to the input images. I am confused that is this specifically designed? Or did I do anything wrong? 
The size of SYNTHIA images is 1280x760. I set the train_crop_size as 513x513. The train_batch_size is 8 , the output_stride is 16 and fine_tune_batch_norm = True. 
I noticed the gray padding would occur in 10% images. As shown in tensorboard, the strange one is like:
![wechatimg146](https://user-images.githubusercontent.com/37647961/40285480-cd987558-5cdf-11e8-8aa8-926c5ad65647.jpeg)
While one good example is like:
![wechatimg145](https://user-images.githubusercontent.com/37647961/40285498-233f87a8-5ce0-11e8-84d0-edda1db1cc36.jpeg)
Hope for suggestions

**Comment by aquariusjay** [comment]
I think you are using the default value of min_scale_factor = 0.5.
https://github.com/tensorflow/models/blob/master/research/deeplab/train.py#L140
The training code dynamically scales the input images with different scale factor values. When scale_factor = 0.5, the image is rescaled to 0.5 * (1280x760) = 640x380 and you have crop_szie = 513x513, resulting in some padded regions.

**Comment by jytime** [comment]
Thank you. There are no paddings now although it still can only achieve an IoU of 43.5%. I will try to find the problem and share the solution if possible.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
